dm/ctl/common/config.go

```diff
@@ -69,54 +75,57 @@ func (c *Config) String() string {
 }
 
 // Parse parses flag definitions from the argument list.
-func (c *Config) Parse(arguments []string) error {
+func (c *Config) Parse(arguments []string) (bool, error) {
```

Comment:
give a variable name for `bool` to make it readable?

Reply:
[Done](https://github.com/pingcap/dm/pull/287/files#diff-05ef9cad41ee61bf4ad94f2d5a544ddaR78)